src/Filesystem/Contracts/FileContract.php

```diff
@@ -5,4 +5,8 @@
 interface FileContract
```

Comment:
Přejmenovat na FileServiceInterface ? minimálně je to interface

Reply:
upraveno